engine/common/api_helpers/filters.py

```diff
@@ -69,6 +70,22 @@ def filter_model_field_with_single_value(self, queryset, name, value):
         queryset = queryset.filter(**lookup_kwargs)
         return queryset
 
+    def filter_model_field_with_multiple_values(self, queryset, name, values):
```

Comment:
adding a method to filter on multiple teams
we reuse the same mixin we use for public api